### PR TITLE
create homework 7

### PR DESCRIPTION
**Pull request by NikGares**

### CODE REVIEW

File: HW7/style.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    margin-bottom: 15px;
+}
+
+.footer_table {
+    width: 100%; 
+    background-color: rgb(255, 255, 255); 
+    table-layout: fixed;
+}
+
+.b1 {
```
Comment (Killerkip10) [inline HW7/style.css]: Not informative class name

File: HW7/style.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    border: 0;
+    width: 20%;
+    height: 50px;
+    float: left;
+    margin: 10px;
+    min-width: 150px;
+    background:rgb(255, 255, 255);
+    text-align: left;
+    border-radius: 5px;
+    text-indent: 5px;
```
Comment (Killerkip10) [inline HW7/style.css]: Redundant, use padding in this case

File: HW7/style.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+.button_course:hover {
+    box-shadow: 2px 2px white, 4px 4px rgb(184, 171, 171);
+    top: -4px;
+    left: -4px;
+}
+
+.button_course:hover:after,
+.button_course:focus:after {
+    width: 100%;
```
Comment (Killerkip10) [inline HW7/style.css:173]: It is not good idea to change width while animate. Use transform for this

**Review by Killerkip10 — COMMENTED** [review]
## Task 1
- (1.6) good animation, but sometimes your animation breaks. (-5 points);

### 75 points

## Task 2
### 20 points

### Total 95 points

File: HW7/style.css (hunk 3 above)
Comment (NikGares) [reply HW7/style.css:173]: Fix this

File: HW7/style.css (hunk 2 above)
Comment (NikGares) [reply HW7/style.css]: Done

File: HW7/style.css (hunk 1 above)
Comment (NikGares) [reply HW7/style.css]: Edited the class name

**Comment by Killerkip10** [comment]
You can merge this

**Pull request merged**